I added a mips machine class with register variabls and a constructor…

alexgarcia723 opened the pull request:
created the start to the machine class with a constructor and a few variables

danielt3131 committed:
Moved MipsMachine.java into machine subpackage

danielt3131 committed:
Fixed IntelliJ config for the java version to 21.

danielt3131 committed:
Git will now ignore .idea/gradle.xml to resolve errors for different machines.

danielt3131 committed:
Removed .idea/gradle.xml from git

danielt3131 committed:
Fixed code style in MipsMachine.java

alexgarcia723 committed:
I added a mips machine class with register variabls and a constructor to declare the amount of memory the machine will have

danielt3131 commented:
I added whitespace for each variable declaration.
The MipsMachine.java is moved into a subpackage called machine.
Fixed IntelliJ configs.
All you need todo is to add javadocs to your class file.
Ignore the incorrect timestamps.
Will accept once javadocs and the license declaration are added to the file.

alexgarcia723 committed:
I added a mips machine class with register variabls and a constructor to declare the amount of memory the machine will have

alexgarcia723 commented:
I added some java doc comments along with some better documentation on what the different registry variables are
I also added some skeleton code for some methods I think we'll need but they can definitely be removed if we decide to do this in a different way.

alexgarcia723 committed:
I added a mips machine class with register variabls and a constructor to declare the amount of memory the machine will have

alexgarcia723 commented:
added the licence
